test/algorithms/set_operations/difference/difference.cpp

```diff
@@ -345,17 +345,18 @@ void test_all()
 #endif
 
     {
-        //geos_2_a #area expected: 138.69238279999999008 detected: 138.53125 type: f
```

Comment:
is this replaced by `expectation_limits` below? Should we keep stating the library and version  (geos?) used to produce the expected results?

Reply:
In the past I incidentally mentioned used libraries or tools, but it's too much work to do that everywhere.
And now that we have a range of allowed values it's kind of redundant.